dm/config/security.go

```diff
@@ -28,8 +29,11 @@ type Security struct {
 	SSLKey        string   `toml:"ssl-key" json:"ssl-key" yaml:"ssl-key"`
 	CertAllowedCN strArray `toml:"cert-allowed-cn" json:"cert-allowed-cn" yaml:"cert-allowed-cn"`
 	SSLCABytes    []byte   `toml:"ssl-ca-bytes" json:"-" yaml:"ssl-ca-bytes"`
```

Comment:
Do we expect users to specify the SSL-related configurations in bytes? If not, we should not expose this configuration items.

Reply:
Yes, currently user can send the bytes by OpenAPI, and after DM is integrated to engine certificate bytes can be loaded from k8s secret.